hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/lock/LockManager.java

```diff
@@ -76,6 +76,7 @@ public void lock() {
             break;
           }
           metrics.updateLockNotAcquiredMetric();
+          LOG.warn("Current lock owner information ： " + lockProvider.getCurrentOwnerLockInfo());
```

Comment:
please use english colon instead of chinese colon. `"Current lock owner information : "`

Reply:
done.